docker/Dockerfile

```diff
@@ -93,5 +93,8 @@ RUN \
 USER node
 ENV NODE_ENV=production
 
+# Expose Ports - Default to 4041 for NORTH PORT, 5684 for LWM2M PORT
+EXPOSE ${IOTA_NORTH_PORT:-4041} ${LWM2M_PORT:-5684}
```

Comment:
docker/README.md docker compose example should be fixed to expose and map 5684:

```
        - "7896"
    ports:
        - "4041:4041"
        - "7896:7896"
```

In addition, maybe it's a good idea to include and describe the LWM2M_PORT environment variable there (as in other agents the HTTP_PORT has been included).

Reply:
fixed d5f0438